scargo/commands/flash.py

```diff
@@ -23,16 +23,16 @@ def scargo_flash(
     config = prepare_config()
     target = config.project.target
 
-    if port and target.family != "esp32":
-        logger.error("--port option is only supported for esp32 projects.")
+    if port and (target.family != "esp32" or target.family != "stm32"):
```

Comment:
`target.family != "esp32" or target.family != "stm32"` is always true.
You should use `and` instead, or `not in`

Reply:
true, corrected